doc/reference/reference_lua/connpool.rst

```diff
@@ -0,0 +1,188 @@
+..  _connpool_module:
+
+Module experimental.connpool
+============================
+
+**Since:** :doc:`3.1.0 </release/3.1.0>`
+
+..  important::
+
+    ``experimental.connpool`` is an experimental module and is subject to changes.
+
+An ``experimental.connpool`` module provides a set of features for connecting to remote cluster instances and for executing remote procedure calls on an instance that satisfies the specified criteria.
```

Comment:
```suggestion
The ``experimental.connpool`` module provides a set of features for connecting to remote cluster instances and for executing remote procedure calls on an instance that satisfies the specified criteria.
```

Reply:
I see that elsewhere in the doc we keep using a definite article before module names, even when introduced for the first time (e.g. https://docs.d.tarantool.io/en/doc/connpool-api/reference/reference_lua/iconv/), with the only exception of explicitly adding "new" like in "a new XXX module" in release notes, but not in the main doc.